src/wp-admin/about.php

```diff
@@ -39,6 +39,37 @@
 			</nav>
 		</div>
 
+		<div class="about__section changelog">
+			<div class="column">
+				<h2><?php _e( 'Maintenance Release' ); ?></h2>
+				<p>
+					<?php
+					printf(
+						/* translators: 1: WordPress version number, 2: plural number of bugs. */
+						_n(
+							'<strong>Version %1$s</strong> addressed %2$s bug.',
+							'<strong>Version %1$s</strong> addressed %2$s bugs.',
+							666
+						),
+						'5.9.1',
+						number_format_i18n( 666 )
```

Comment:
@audrasjb is the number right? Just checking to make sure before it's committed.

Reply:
no, that's a placeholder, I'll update it sooner :)

Reply:
82 is the correct number.